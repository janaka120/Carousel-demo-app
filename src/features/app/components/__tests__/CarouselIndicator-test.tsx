import React from 'react';
import renderer from 'react-test-renderer';
import CarouselIndicator from '../CarouselIndicator';

test('CarouselIndicator renders correctly', () => {
  const tree = renderer
    .create(<CarouselIndicator length={4} selectedIndex={0} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
