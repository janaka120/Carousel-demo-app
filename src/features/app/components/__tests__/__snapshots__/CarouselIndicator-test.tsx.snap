// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarouselIndicator renders correctly 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "paddingHorizontal": 10,
      "paddingVertical": 9.857142857142858,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "borderColor": "#DED9D9",
        "borderRadius": 9.107142857142858,
        "borderWidth": 1.8214285714285714,
        "height": 19.714285714285715,
        "marginHorizontal": 4,
        "width": 19.714285714285715,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#DED9D9",
        "borderColor": "#DED9D9",
        "borderRadius": 9.857142857142858,
        "borderWidth": 1.8214285714285714,
        "height": 19.714285714285715,
        "marginHorizontal": 4,
        "width": 19.714285714285715,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#DED9D9",
        "borderColor": "#DED9D9",
        "borderRadius": 9.857142857142858,
        "borderWidth": 1.8214285714285714,
        "height": 19.714285714285715,
        "marginHorizontal": 4,
        "width": 19.714285714285715,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#DED9D9",
        "borderColor": "#DED9D9",
        "borderRadius": 9.857142857142858,
        "borderWidth": 1.8214285714285714,
        "height": 19.714285714285715,
        "marginHorizontal": 4,
        "width": 19.714285714285715,
      }
    }
  />
</View>
`;
